packages/swirl-components/src/components/swirl-card/swirl-card.tsx

```diff
@@ -24,6 +25,15 @@ export class SwirlCard {
       "card--interactive": this.interactive || this.href,
     });
 
+    /**
+     * Idea:
+     * - media section above
+     * - content section below
+     * - use flex gap
+     * - swirl stack
+     * ::slotted(*:img)
+     */
+
     return (
       <Host>
         <Tag
```

Comment:
**issue:** When used as a link, the card contents would be too much for an accessible label. You can add an optional swirlAriaLabel prop for these cases.

```ts
@Prop() swirlAriaLabel?: string;
```

```jsx
<Tag
  aria-label={this.swirlAriaLabel}
  class={className}
  href={this.href}
```